client/src/app.ts

```diff
@@ -4,9 +4,10 @@ import { ethers } from 'ethers';
 import { SignClient } from '@walletconnect/sign-client';
 import { SafeInfo, NetworkConfig, Token } from './types';
 import { truncateAddress, resolveEnsName, prepareTransactionRequest, calculateSafeTxHash, formatSafeSignatures, getSafeNonce, getSafeTxHashFromContract } from './utils';
-import { NETWORKS, DEFAULT_NETWORK, getNetworkConfig, COMMANDS, getContractAddress, getExplorerUrl } from './config';
+import { NETWORKS, DEFAULT_NETWORK, getNetworkConfig, getContractAddress, getExplorerUrl } from './config';
 import { SafeTxPool } from './managers/transactions';
 import { PriceOracle } from './services/oracle';
+import { HelpGuide } from './components/HelpGuide';
```

Comment:
I think we will put index.ts file into 'components' to make it become a module because we might have more other components later